crates/taplo-cli/src/args.rs

```diff
@@ -22,6 +22,8 @@ pub struct TaploArgs {
 #[derive(Clone, Args)]
 pub struct GeneralArgs {
     /// Path to the Taplo configuration file.
+    ///
+    /// Configuration file can be specified in $TAPLO_CONFIG
     #[clap(long, short)]
```

Comment:
`clap` already supports this feature. Let's remove the added documentation (it is added by clap), add the `env` feature to the `clap` dependency, and update the code to:

```suggestion
    #[clap(long, short, env = "TAPLO_CONFIG")]
```